common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -45,6 +45,10 @@ class @Problem
     @saveNotification = @$('.notification-save')
     @saveButtonLabel = @$('.action .save .save-label')
     @saveButton.click @save
+    @gentleAlertNotification = @$('.notification-gentle-alert')
+
+    # Hide the Gentle Alert notification until needed
+    @gentleAlertNotification.hide()
```

Comment:
@staubina I still have questions on why it is necessary to call hide initially (vs. just having display value of none).